operations/hpe_pdu/hpe_pdu_admin_procedures.md

```diff
@@ -0,0 +1,119 @@
+3. Connect to the HPE PDU Web Interface (See [Connect to HPE PDU Web Interface](#connect-to-hpe-pdu-web-interface)) and log in as `admin`.
+4. Using the **"Settings"** Icon (gear in computer monitor in top right corner) navigate to **"System Management"**
+5. Click the **"Update Firmware"** button.
+6. Click **"Choose File"** and select the **"HPE.FW"** file downloaded.
+7. Click **"Upload"** button.
+
+The firmware will be updated and the PDU management processor will restart.
+
+#### Change HPE PDU User Passwords
+1. Connect to the HPE PDU Web Interface (See [Connect to HPE PDU Web Interface](#connect-to-hpe-pdu-web-interface)) and log in as `admin`.
+2. Using the **"admin"** menu (top right corner) and navigate to **"User Accounts"**.
+3. Click on the **"Edit"** icon (pencil) next to the user.
+4. Enter new password and make any other changes for that user account and click the **"Save"** button.
+
+#### Discover HPE PDU after Upgrading CSM
+**This procedure is only needed when upgrading CSM, not performing a fresh install.  This procedure should be run after CSM has been fully upgraded including the discovery job.**
+1.  In CSM 1.0 and before the hms-discovery job and Redfish Translation Service (RTS) treated all PDUs as if were made by ServerTech, and after the upgrade to CSM 1.2 RTS will still think the HPE PDUs in the system are ServerTech PDUs. These erroneous HPE PDU entries for RTS need to get removed from Vault.  
+    ```bash
+    ncn-m001# VAULT_PASSWD=$(kubectl -n vault get secrets cray-vault-unseal-keys -o json | jq -r '.data["vault-root"]' |  base64 -d)
+    ncn-m001# alias vault='kubectl -n vault exec -i cray-vault-0 -c vault -- env VAULT_TOKEN=$VAULT_PASSWD VAULT_ADDR=http://127.0.0.1:8200 VAULT_FORMAT=json vault'
+    ```
+    1.  Identify HPE PDUs that RTS is aware of:
+        ```bash
+        ncn-m001# vault kv list secret/pdu-creds
+        [
+          "global/",
+          "x3000m0",
+          "x3000m1"
+        ]
+        ```
+    2.  For each HPE PDU Identified in the command above remove it from Vault:
+        ```bash
+        ncn-m001# PDU=x3000m0
+        ncn-m001# vault kv delete secret/pdu-creds/$PDU
+        ```
+    3.  Restart the Redfish Translation Service (RTS):
+        ```bash
+        ncn-m001# kubectl -n services rollout restart deployment cray-hms-rts
+        ncn-m001# kubectl -n services rollout status deployment cray-hms-rts
+        ```
+2. Find the list of PDU MAC address.  The `ID` field in each element is the normalized MAC address of each PDU:
+    ```bash
+    ncn-m001# cray hsm inventory ethernetInterfaces list --type CabinetPDUController
+    ```
+3. Use the `"ID"` from the command above to delete each HPE PDU MAC address from HSM:
+    ```bash
+    ncn-m001# cray hsm inventory ethernetInterfaces delete {ID}
+    ```
+4. On the next `hms-discovery` job run, it should locate the PDU and discover it correctly as a HPE PDU.

```

Comment:
This isn't really a step as no action is required. Can the text from this line be added to the previous step?

Reply:
Added it to the next step.